cmd/utils/flags.go

```diff
@@ -268,6 +270,17 @@ var (
 		Usage:    "Manually specify the Verkle fork timestamp, overriding the bundled setting",
 		Category: flags.EthCategory,
 	}
+	PathBasedSchemeFlag = &cli.BoolFlag{
+		Name:     "trie.path-based",
+		Usage:    "Enables experiment path-based state scheme (default = disabled)",
+		Category: flags.MiscCategory,
+	}
+	StateHistoryFlag = &cli.Uint64Flag{
```

Comment:
Out of curiosity, is there a specific reason to make this configurable?

Reply:
Because for archive node, you want to reserve more histories, but only the latest 90K.

Reply:
But we already have the gcmode flag that people rely on. Wouldn't it be simpler to simply use that to retain all history and otherwise retain 90K?

Reply:
But with this flag, you will have more flexibilities, you can control the length of history, not just keep all or keep 90K.

But I guess if you want to maintain an archive node, you just simply want all histories?